src/algorithm.rs

```diff
@@ -61,3 +61,21 @@ impl From<Algorithm> for String {
         algorithm.to_string()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn algorithms_serialize_and_deserialize_from_display_form() {
```

Comment:
```suggestion
    fn serialize_and_deserialize_from_display_form() {
```

Since this is now inside the `algorithm` module, the prefix can be removed.